README.md

```diff
@@ -10,8 +10,10 @@
 
 Cloud Health Connect provides a Connect Middleware for use in Express.js, Loopback and other frameworks that uses [Cloud Health](http://github.com/CloudNativeJS/cloud-health) to provide:
 
+* Combined Health checks
```

Comment:
is this duplication?  
`Combined Health (Readiness and Liveness) checks`...

Reply:
Removed first entry.